packages/paste-website/src/components/ThemeSwitcher.tsx

```diff
@@ -1,13 +1,77 @@
 import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
 import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Box} from '@twilio-paste/box';
+import {Themes} from '../constants';
+import {useSiteTheme, SiteTheme} from '../context/SiteThemeContext';
 
 interface ThemeSwitcherProps {
   children?: React.ReactElement;
 }
+
+const StyledThemeSwitcherLabel = styled.label<{}>(props => ({
+  cursor: 'pointer',
+  display: 'inline-block',
+  padding: `${themeGet('space.space20')(props)} ${themeGet('space.space40')(props)}`,
+  '&:hover': {
+    textDecoration: 'underline',
+  },
+}));
+
+const StyledThemeSwitcherRadio = styled.input<{}>(props => ({
+  opacity: 0,
+  position: 'absolute',
```

Comment:
using opacity and pos absolute instead of a more traditional "off screen" solution due to bug with Dragon Naturally speaking (most common dictation software) that we found at Salesforce where it couldn't find checkboxes or radio buttons if they were visually hidden any other way than opacity.

Reply:
Good to know.